netket/operator/_kinetic.py

```diff
@@ -60,18 +63,22 @@ def __init__(
         hilbert: AbstractHilbert,
         mass: Union[float, list[float]],
         dtype: Optional[DType] = None,
+        method: str = "parallel",
     ):
         r"""Args:
         hilbert: The underlying Hilbert space on which the operator is defined
         mass: float if all masses are the same, list indicating the mass of each particle otherwise
         dtype: Data type of the matrix elements. Defaults to `np.float64`
+        method: Method to compute the kinetic energy: "sequential" or "parallel", where sequential uses less memory
```

Comment:
I would either rename these to "vmap" and "forloop" or something similar or say what is actually meant with 'sequential' and 'parallel'. Smth like: "Sequential" uses a jax.fori_loop over all tangent vectors to compute the diagonal of the hessian while "parallel" uses jax.vmap. "Sequential" therefore uses less memory.

Reply:
@PhilipVinc , what do you think?

Reply:
I would be more inclined to think this should be two different flags.
One is the algorithm and the other is the chunk size (which does not necessarily need to be 1).

For example if you want to use `Feenberg` you might want or not to chunk it as well.
Or if you want to use Taylor mode AD or Forward mode Laplacian... those are all methods, and they might or might require chunking.

NOTE: by chunking here of course I'm referring to the chunking of the degrees of freedom. So to avoid confusion the correct name should not be chunking...

Reply:
Ok I agree with @PhilipVinc but I would only provide as algorithm: Feenberg, Taylormode, Normal
and then chunking = bool. If True: foriloop otherwise vmap.
I would not provide any algorithm with fwd-mode for the first derivative or reverse-mode for the second derivative. As far as I understand you want to use reverse mode if f:R^n -> R^m has m << n. For us we will always have m < n if not m<<n because m=1 and n = Nparticles * dim. 
Then for df: R^n -> R^(nxn) does not have m < n so you always want to use forward mode. Do I understand something wrong?

Reply:
Yes indeed. The fwd fwd mode is not useful at all for our use cases